diff --git a/node_modules/@types/url-parse/index.d.ts b/node_modules/@types/url-parse/index.d.ts
index 8033adc..6c26e26 100755
--- a/node_modules/@types/url-parse/index.d.ts
+++ b/node_modules/@types/url-parse/index.d.ts
@@ -25,7 +25,7 @@ declare namespace URLParse {

     type QueryParser<T = Record<string, string | undefined>> = (query: string) => T;

-    type StringifyQuery = (query: object) => string;
+    type StringifyQuery = (query: object, prefix: boolean) => string;
 }

 interface URLParse<Query> {
